aws/lambda-with-other-services/send-sms-code-with-sns/send-sms-code-with-sns.md

```diff
@@ -0,0 +1,123 @@
+# Using SNS and Lambda to Send a Random 6 Digit Number Via Text Message
+
+### By Tim Petrone
+
+AWS SNS is a cloud based web service that allows you to send and recieve SMS messages to individual or multiple devices. You can send text messages to mobile phones or SMS-enabled deivces. (e.g. Amazon Kindle) In this documentation, you will learn how to send a text message to an individual mobile phone with an auto generated verification code as the message.
+
+To start, create a lambda function and require the AWS software development kit at the top like so: `const AWS = require('aws-sdk');`. This allows for the use of Amazon Web Services' various platforms. In this case, we will be using AWS.SNS.
```

Comment:
Might be better just to create the whole handler in its simplest form up to this point, like so:

    const AWS = require('aws-sdk');

    module.exports.handler = function(event, context, callback) {
        const SNS = new AWS.SNS();

        //...
    }

After that, you can explain each part. As time goes on, just add JS code blocks, explaining that they go after the last line within your function.